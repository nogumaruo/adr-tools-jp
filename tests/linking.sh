adr new First Record
adr new Second Record
adr new Third Record
adr link 3 Amends 1 "Amended by"
adr link 3 Clarifies 2 "Clarified by"
head -14 doc/adr/001-first-record.md
head -14 doc/adr/002-second-record.md
head -16 doc/adr/003-third-record.md
