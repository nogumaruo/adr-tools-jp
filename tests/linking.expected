adr new First Record
doc/adr/001-first-record.md
adr new Second Record
doc/adr/002-second-record.md
adr new Third Record
doc/adr/003-third-record.md
adr link 3 Amends 1 "Amended by"
adr link 3 Clarifies 2 "Clarified by"
head -14 doc/adr/001-first-record.md
# 1. First Record

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中

Amended by [3. Third Record](003-third-record.md)

## コンテキストと問題点の説明

head -14 doc/adr/002-second-record.md
# 2. Second Record

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中

Clarified by [3. Third Record](003-third-record.md)

## コンテキストと問題点の説明

head -16 doc/adr/003-third-record.md
# 3. Third Record

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中

Amends [1. First Record](001-first-record.md)

Clarifies [2. Second Record](002-second-record.md)

## コンテキストと問題点の説明

